src/mongocxx/collection.cpp

```diff
@@ -520,23 +520,28 @@ stdx::optional<bsoncxx::document::value> collection::find_one_and_replace(
     auto opts = libmongoc::find_and_modify_opts_new();
     auto opts_cleanup = make_guard([&opts] { libmongoc::find_and_modify_opts_destroy(opts); });
     int flags = ::MONGOC_FIND_AND_MODIFY_NONE;
-    scoped_bson_t bson_replacement{replacement};
-    scoped_bson_t bson_sort{options.sort()};
-    scoped_bson_t bson_projection{options.projection()};
```

Comment:
Hmm, I'm having trouble understanding your position here.  If glancing at this change makes you uncertain about its correctness, then to me it doesn't seem appropriate at all to commit it as an unreviewed minor change.  Note also that this change is committed separately in this review (8c213c335a2549452e56e4171709ad4e83374870).  Perhaps let's discuss further in person.